dist/scripts/vendor.js

```diff
@@ -8062,7 +8062,7 @@ delete c[b.id], delete d[a];
 http:80,
 https:443,
 ftp:21
-}, sf = b("$location"), tf = /^\s*[\\/]{2,}/, uf = {
+}, sf = b("$location"), tf = /^\s*[\\\/]{2,}/, uf = {
```

Comment:
This also looks incorrect :/ What node version do you have? Unfortunately anything newer than Node 4 generates slightly different dist output, which will cause merges to fail.

Probably easiest for now is to 

```
$ git checkout master dist/scripts/vendor.js
```

since you only have markup changes.